src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task {
+
+    protected String by;
+    protected boolean isDone;
+    protected String category;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        this.isDone = false;
```

Comment:
Unnecessary to use "this" in reference to the properties isDone and category when there are no similarly named arguments passed into the method.

Reply:
Thanks for your feedback !